apps/minifront/src/state/staking/index.ts

```diff
@@ -170,15 +167,49 @@ const byBalanceAndVotingPower = (valueViewA: ValueView, valueViewB: ValueView):
   return byVotingPower;
 };
 
+export const { stakingTokensAndFilter, useStakingTokensAndFilter } = createZQuery({
+  name: 'stakingTokensAndFilter',
+  fetch: async () => {
+    const balancesByAccount = await getBalancesByAccount();
```

Comment:
random question out of curiosity, can you somehow _chain_ zquery requests? for instance, instead of calling `getBalancesByAccount()`, something like

```
    // Access the state of balancesByAccount
    const balances = useStore.getState().balances.balancesByAccount;
```

Reply:
don't have specific use case in mind, was just curious if this was possible.